runtime/src/iree/hal/drivers/vulkan/command_queue.h

```diff
@@ -20,6 +20,32 @@ namespace iree {
 namespace hal {
 namespace vulkan {
 
+// A single batch of command buffers submitted to a device queue.
+// All of the wait semaphores must reach or exceed the given payload value prior
+// to the batch beginning execution. Each command buffer begins execution in the
+// order it is present in the list, though note that the command buffers
```

Comment:
nit: plural for "payload values" to avoid confusion about there being a single value shared across all semaphores. That also matches the plural used below for signal semaphores.
```suggestion
// A single batch of command buffers submitted to a device queue.
// All of the wait semaphores must reach or exceed the given payload values
// prior to the batch beginning execution. Each command buffer begins execution
// in the order it is present in the list, though note that the command buffers
```